lua/neotest-vitest/init.lua

```diff
@@ -16,10 +16,10 @@ local adapter = { name = "neotest-vitest" }
 ---@param path string
 ---@return boolean
 local function hasVitestDependency(path)
-  local rootPath = util.find_package_json_ancestor(path)
+  local rootPath = lib.files.match_root_pattern("package.json")(path)
```

Comment:
I do not understand the reason for change this line. I'm afraid we are not going support the case of monorepo

Reply:
I tried this code and it seems to work also in a monorepo. I resolve this comment